site/components/event-title/index.js

```diff
@@ -1,21 +1,22 @@
-import React, { PropTypes } from 'react';
+// @flow
+import * as React from 'react';
 import styles from '../../pages/events/events-list/style.css';
 import Link from '../link';
 
-const EventTitle = ({ eventLink, eventTitle }) => (
+type EventTitleProps = {
+  eventLink: {
+    to?: string,
+    children?: React.Node,
```

Comment:
It looks like these might be wrong? Does `Link` work correctly if those values are missing?